app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,11 @@
+class LikesController < ApplicationController
+  def create
+    @like = Like.new(author_id: params[:user_id], post_id: params[:post_id])
```

Comment:
- [ ] @NicolasAndrehh great work on your  create method, however, we are supposed to use the currently logged-in user (current_user) of the app using the `current_user` method we created,
`hint`

```suggestion
      @post = Post.find(params[:post_id])
      @like = @current_user.likes.new(post_id: @post.id)
```